hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -865,20 +885,23 @@ static int xpadneo_input_configured(struct hid_device *hdev, struct hid_input *h
 
 	if (param_gamepad_compliance) {
 		hid_info(hdev, "enabling compliance with Linux Gamepad Specification\n");
-		abs_min = -32768;
-		abs_max = 32767;
+    deadzone = 0;
+		/* abs_min = -32768; */
+		/* abs_max = 32767; */
+    abs_min = -127;
```

Comment:
If your model reports 0..255 as the axis range, you need to switch cases here for your model. But I rather suggest to instead rescale to the Xbox controller range and patch the HID reports accordingly because we patch the VID/PID during device discovery. Otherwise your changes won't match games expectations.